test/Test/SAML2/WebSSO/XMLSpec.hs

```diff
@@ -54,15 +54,15 @@ spec = describe "XML Sanitization" $ do
         `shouldBe` (emailNameID "somebody@example.org" :: Either String NameID)
 
     it "should fail to decode an invalid email" $ do
-      decodeElem @NameID (xmlWithName emailFormat "&lt;somebody@example.org&gt;")  
+      decodeElem @NameID (xmlWithName emailFormat "&lt;somebody@example.org&gt;")
```

Comment:
@ChrisPenner if you like, you can teach your vim to do that for us.  If not, don't bother, I have "write an integration test for this" somewhere on a TODO list.  ;-)

Reply:
Sorry, what are you referring to? trailing whitespace issues or something else?

Reply:
> Sorry, what are you referring to? trailing whitespace issues or something else?

sorry, yes.  trailing whitespace issues.